### PR TITLE
Update ug

### LINKED ISSUE #281: Add switch command and exit command images for UG

### PR DESCRIPTION
**Pull request by seanjyjy**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -559,14 +559,31 @@ There are three `TYPE`s:
 
 `com` refers to Company tab, `app` refers to Application tab, `me` refers to Profile tab.
 
-Example: 
-* `switch me`
+Example:
+
+1. Suppose you want to switch to the application tab. Executing `switch app` will switch the tabs to application tab.
```
Comment (orzymandias) [inline docs/UserGuide.md]: Maybe you could change it to switch tabs to the application tab

File: docs/UserGuide.md [hunk 2]
```diff
@@ -559,14 +559,31 @@ There are three `TYPE`s:
...
+
+<p align="center"><img src="images/ug-general/switchApp.png" width="70%" height="70%"/></p>
+
+2. Note that the tabs has been changed as well as the cards and display.
```
Comment (orzymandias) [inline docs/UserGuide.md]: Perhaps you can update it to *have been changed

File: docs/UserGuide.md [hunk 3]
```diff
@@ -575,12 +592,45 @@ Format: `help`
 <div markdown="span" class="alert alert-primary">
   :bulb: <strong>Tip:</strong> You can press Esc key to close the help window!
 </div>
+<br/>
+
+Example:
+
+1. Suppose you can't remember the commands. Fret not, executing `help` will generate a new window containing the link to your user guide!
```
Comment (orzymandias) [inline docs/UserGuide.md]: *the user guide? since your implies the user owns the guide?

**Review by orzymandias — COMMENTED** [review]
Minor nit, Everything else lgtm

File: docs/UserGuide.md (hunk 3 above)
Comment (seanjyjy) [reply docs/UserGuide.md]: fixed

File: docs/UserGuide.md (hunk 2 above)
Comment (seanjyjy) [reply docs/UserGuide.md]: fixed

File: docs/UserGuide.md (hunk 1 above)
Comment (seanjyjy) [reply docs/UserGuide.md]: fixed

**Review by orzymandias — APPROVED** [review]
LGTM

**Pull request merged**